src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java

```diff
@@ -100,22 +100,30 @@ public CftQueryService(CamundaService camundaService,
         //1. Evaluate Dmn
         final List<Map<String, CamundaVariable>> evaluateDmnResult = camundaService.evaluateTaskCompletionDmn(
             searchEventAndCase);
+        log.info("RWA-1172 evaluateDmnResult: {}", evaluateDmnResult);
 
         // Collect task types
         List<String> taskTypes = extractTaskTypes(evaluateDmnResult);
+        log.info("RWA-1172 evaluateDmnResult.size: {}", evaluateDmnResult.size());
+        log.info("RWA-1172 taskTypes.size: {}", taskTypes.size());
+        log.info("RWA-1172 taskTypes: {}", taskTypes);
```

Comment:
I have a habbit of wrapping values in square brackets [] 
e.g.         log.info("RWA-1172 evaluateDmnResult: [{}]", caseId);
As it highlighted a trailing spaces issue before. Up to you if you want to bother, but I find it useful